tests/network/vmi_networking.go

```diff
@@ -483,7 +483,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			dhcpVMI.Spec.Domain.Devices.Interfaces[0].DHCPOptions = &v1.DHCPOptions{
 				BootFileName:   "config",
 				TFTPServerName: "tftp.kubevirt.io",
-				NTPServers:     []string{"127.0.0.1", "127.0.0.2"},
+				NTPServers:     []string{"162.159.200.0123", "010.35.255.14"},
```

Comment:
Why did you change the original loopback addresses?
Using real addresses may cause traffic to exit the test setup and result in unexpected behavior. We not always know what is the test setup (e.g. kubevirtci is only one of them).

Reply:
Reverted to original IP addresses.